### PR TITLE
use find_flexlm_license function in IntelBase generic easyblock

### LINKED ISSUE #555: when INTEL_LICENSE_FILE env var points to a dir, append every license in dir
now if INTEL_LICENSE_FILE is pointing to a directory, just the first license in the directory is used.

EB should do something like INTEL_LICENSE_FILE="LIC1:LIC2:LIC3"

See https://github.com/hpcugent/easybuild-easyblocks/blob/master/easybuild/easyblocks/generic/intelbase.py#L236

### PR DESCRIPTION
**Pull request by boegel**
requires ~~https://github.com/hpcugent/easybuild-framework/pull/1633~~

see discussion in #837  (cc @damianam, @valtandor)

also fixes #555 (cc @pescobar)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyblocks/generic/intelbase.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 from easybuild.framework.easyblock import EasyBlock
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.filetools import read_file
+from easybuild.tools.filetools import find_flexlm_license, read_file
```
Comment (damianam) [inline easybuild/easyblocks/generic/intelbase.py]: I think you don't really need read_file

Comment (boegel) [inline easybuild/easyblocks/generic/intelbase.py]: Still needed, in `parse_components_list`

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyblocks/generic/intelbase.py [hunk 2]
```diff
@@ -208,82 +208,30 @@ def configure_step(self):
...
-                                     "something went horribly wrong...")
+        if lic_specs:
+            if self.license_env_var is None:
+                self.log.info("Using Intel license specifications from 'license_file': %s", lic_specs)
+                self.license_env_var = default_lic_env_var
+            else:
+                self.log.info("Using Intel license specifications from $%s: %s", self.license_env_var, lic_specs)
 
+            self.license_file = os.pathsep.join(lic_specs)
             self.cfg['license_file'] = self.license_file
```
Comment (boegel) [inline easybuild/easyblocks/generic/intelbase.py]: not needed?

cc @damianam

**Comment by boegel** [comment]
retested after last change with all Intel tools, good to go

Thanks for the feedback @wpoely86 and @damianam!

**Pull request merged**